sample_package_configs/adot/daemonset_default/daemonset_default.yaml

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
The reason why I'm creating separate folders here is because I want to add rendered results later in each folder as well.